debian/xenial/kubeadm/debian/rules

```diff
@@ -11,7 +11,7 @@ binary:
 	#curl  --fail -sS -L \
 	#	-o usr/bin/kubeadm \
 	#	"https://storage.googleapis.com/kubernetes-release/release/v{{ .Version }}/bin/linux/{{ .Arch }}/kubeadm"
-	printf '#!/bin/bash\necho "this is kubeadm"' > usr/bin/kubeadm
+	cat kubeadm-e7c8aa5.base64 | base64 --decode > usr/bin/kubeadm
```

Comment:
+1 to abstracting to files. But do you want to rely on the hash in the filename or instead title it `kubeadm.base64` so it's less likely to require changes on the next iteration of the package?